gradle/publishing.gradle

```diff
@@ -19,6 +19,11 @@ plugins.withType(MavenPublishPlugin) {
                         developerConnection = 'scm:git:ssh://git@github.com/reactor/BlockHound.git'
                     }
                     developers {
+                        developer {
```

Comment:
@bsideup should I change your email address below to your gmail one?

Reply:
yes, please.

Reply:
will do in next PR